package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "redesigned-webpack",
+  "version": "1.0.0",
+  "description": "A webpack template for future use",
+  "main": "index.js",
+  "scripts": {
```

Comment:
- [ ] The _script option_ of your _package.json_ file is **not properly configured** and hence your application cannot start using the command `npm start`. Kindly add the necessary _script options_ similar to as can be seen below:

**Set package.json script options:**
![image](https://user-images.githubusercontent.com/12745474/134920809-70aa4e59-cbd7-45db-a19d-cd1f8df37bf8.png)


**Incomplete script configuration:**
![image](https://user-images.githubusercontent.com/12745474/134921410-446afc1e-38d6-4a04-8d81-a69561e319fe.png)